static-webp/src/main/jni/static-webp/webp.cpp

```diff
@@ -438,15 +438,12 @@ jobject WebPImage_nativeGetFrame(JNIEnv* pEnv, jobject thiz, jint index) {
     return nullptr;
   }
 
-
-  auto spIter = std::unique_ptr<WebPIterator, decltype(&WebPDemuxReleaseIterator)> {
-    new WebPIterator(),
-    WebPDemuxReleaseIterator
-  };
+  WebPIterator iter;
 
   // Note, in WebP, frame numbers are one-based.
-  if (!WebPDemuxGetFrame(spNativeContext->spDemuxer->get(), index + 1, spIter.get())) {
+  if (!WebPDemuxGetFrame(spNativeContext->spDemuxer->get(), index + 1, &iter)) {
     throwIllegalStateException(pEnv, "unable to get frame");
+    WebPDemuxReleaseIterator(&iter);
```

Comment:
We should also call `WebPDemuxReleaseIterator(&iter);` in the `if (!spFrameNativeContext)` branch in lines 451-453